app/src/main/java/mobile/programming/musicApp/dto/Instrument.kt

```diff
@@ -1,11 +1,11 @@
-package mobile.programming.musicapp.dto
+package mobile.programming.musicApp.dto
 
 
 data class Instrument(var id: Int, var name: String, var type: String) {
 
     override fun toString(): String{
 
-        return (id.toString() + " " + name + " " + type)
+        return ("$id $name $type")
```

Comment:
Technical Suggestion #2: According to Kotlin coding conventions, the preferred method of combining strings like this is to use string templates (which is what I have changed to in the code) as opposed to string concatenation (which is what you are using). This information can be found here under the "Using strings" segment: https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments.